src/App.js

```diff
@@ -30,7 +36,23 @@ if (token) {
 
 const App = () => (
   <Provider store={store}>
-    <div>Niaje wasee ndio hii hapa!</div>
+    <BrowserRouter>
+      <Container className="bg-gray">
+        <Header/>
+        <Switch>
+          {/* <Route exact path="/" component={HomePage}/> */}
```

Comment:
Please remove all instances of commented code

Reply:
Done.